### PR TITLE
Update volunteer  status from active to lead 

### PR DESCRIPTION
**Pull request by deepankar-sajwan**
The PR contains the code changes for updating user from active volunteer to Lead volunteer,

### CODE REVIEW

File: play-wright/e2e/active-to-lead-volunteer.spec.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import { test, expect } from '@playwright/test';
+import { userDetails, userLogin, submitVolunteerRegistrationForm, searchAndVerifyContact  } from '../utils.js';
+import { VolunteerProfilePage } from '../pages/volunteer-profile.page';
+
+test('update status of volunteer from Active to Lead', async ({ page }) => {
```
Comment (Satendra-SR) [inline play-wright/e2e/active-to-lead-volunteer.spec.js:5]:
```suggestion
test('Add a volunteer to Lead Volunteer group', async ({ page }) => {
```

**Review by Satendra-SR — COMMENTED** [review]
@deepankar-sajwan A few comments, Else looks good. Please fix and merge.

**Review by Satendra-SR — APPROVED** [review]

File: playwright.config.js [hunk 2]
```diff
@@ -14,15 +14,15 @@ require('dotenv').config({path: '.env'});
...
+  timeout: 240 * 1000,
   /* Run tests in files in parallel */
   fullyParallel: true,
   /* Fail the build on CI if you accidentally left test.only in the source code. */
   forbidOnly: !!process.env.CI,
   /* Retry on CI only */
   retries: process.env.CI ? 2 : 0,
   /* Opt out of parallel tests on CI. */
-  workers: process.env.CI ? 6 : 6,
+  workers: process.env.CI ? 9 : 9,
```
[the remark this answers is not in the document]
Comment (deepankar-sajwan) [reply playwright.config.js:25]: CI variable is the default environment variable in GitHub actions (https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables). It is set to true as default. 
We can use different configurations with CI variables to execute the test like workers: process.env.CI ? 6 : 6 for exiting test case with 6 worker threads.

workers: process.env.CI ? 9 : 9 The first option is for worker thread count as 9 for test exeution execution in GitHub actions, and next  option is for worker thread count 9 for test execution in the local environment.  The execution of test cases also take long time in local environment, hence have set higher value for faster execution.

File: play-wright/e2e/active-to-lead-volunteer.spec.js (hunk 1 above)
Comment (deepankar-sajwan) [reply play-wright/e2e/active-to-lead-volunteer.spec.js]: updated tets case description

**Pull request merged**